knowledge-base/pdfprocessing-create-table-of-contents-page.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Create Table of Contents (TOC)
+description: This knowledge base article describes how import PDF documents, merge them, and create a Table of Contents pointing to the merged document pages
```

Comment:
Do not forget the period at the end, the build adds additional content and will look strange without the period.
This knowledge base article describes how **to** import PDF documents...